src/plugins/data/server/autocomplete/value_suggestions_route.ts

```diff
@@ -104,8 +104,8 @@ async function getBody(
   boolFilter: Filter[] = []
 ) {
   const isFieldObject = (f: any): f is IFieldType => Boolean(f && f.name);
-
-  // https://www.opensearch.org/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html#_standard_operators
+  
+  // See https://opensearch.org/docs/latest/opensearch/query-dsl/index/
```

Comment:
Would the correct link here be `https://opensearch.org/docs/latest/opensearch/query-dsl/term/#regex`?